### DISCUSSION TITLE
求助：想润去北欧，国内艺术学士马来西亚MBA硕士，求方法

### DISCUSSION THREAD

**Discussion by Shawnliu66** [The-Run-Philosophy-Organization/run#591] [Q&A]
个人情况，98年出生，存款不到6位数，英语还不错，目前马来西亚国立在读MBA（QS 129）本科是国内摄影系，详细翻看了一下指南，没看到特别详细的欧盟国家润学指南，特意打扰各位寻求帮助。有没有可能拿项目经历去德国法国读二硕或者之类的？希望大家指点一二或者分享指南，谢谢！

**Comment by wocoder** [comment]
[隆重介绍](https://github.com/MarionNoel/juzhanggongzi/issues/39)
[好心安排](https://github.com/MarionNoel/juzhanggongzi/issues/32)
